.github/workflows/bench.yml

```diff
@@ -68,7 +68,7 @@ jobs:
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       - uses: ./.github/actions/bench
-        if: ${{ matrix.target.only_no_opt == 'false' }}
+        if: ${{ !matrix.target.only_no_opt }}
```

Comment:
Are you sure this is going to work? I always get confused what is a string and what is a boolean working with GH actions.

Reply:
It wasn't working before - this step was never run - that's why I changed it.